web/rootfs/defaults/settings-config.js

```diff
@@ -11,6 +11,7 @@
 {{ $ENABLE_WELCOME_PAGE := .Env.ENABLE_WELCOME_PAGE | default "true" | toBool -}}
 {{ $ENABLE_CLOSE_PAGE := .Env.ENABLE_CLOSE_PAGE | default "false" | toBool -}}
 {{ $ENABLE_RECORDING := .Env.ENABLE_RECORDING | default "false" | toBool -}}
+{{ $ENABLE_SERVICE_RECORDING := .Env.ENABLE_SERVICE_RECORDING | default "true" | toBool -}}
```

Comment:
Make the default be ENABLE_RECORDING please.

Reply:
Done!

Note that this change I performed (setting default value for ENABLE_SERVICE_RECORDING to `ENABLE_RECORDING` instead of `true`) does not change anything.

ENABLE_RECORDING is a global flag and setting ENABLE_RECORDING to false will removes the generation of the code for ENABLE_SERVICE_RECORDING.